README.md

```diff
@@ -0,0 +1,123 @@
+# API Specification Version Maven Plugin
+
+The API Specification Version Plugin provides goals to set properties for manifest file OSGI headers and to verify consistence of specification, API and API implementation package names and versions.
+
+**Goals:**
+* `set-spec-properties` (validate phase)
+* `check-module` (package phase)
+
+## Configuration properties
+
+| Property | Type | Default<br/>Value | Description |
+| --- | --- | --- | --- |
+| specMode | `jakarta`&vert;`javax` | `jakarta` | `jakarta`: Jakarta EE projects mode<br/>`javaee`: legacy mode for java.net projects (deprecated) |
+| ignoreErrors | boolean | false | whether `check-module` target should just print warnings or fail the build on errors |
+| spec.nonFinal | boolean | false | `true` for snapshot build, `false` for release build
```

Comment:
The goals actually each have a spec parameter.
The spec parameter is not a value since it's an object. The Maven configuration mapping makes it look like different parameters but it's just that some of the spec attributes are relevant to different goals.

This table makes it look more complicated than it actually is.
I would state the exact goal parameters as declared in each class, and I would document the spec object.

Reply:
Spec object description was left at the beginning of the document.
Each goal has list of parameters based on content of its Mojo class with spec as whole object.